projects/developers.yubico.com/yubikey-manager/package.yml

```diff
@@ -0,0 +1,30 @@
+distributable:
+  url: https://github.com/Yubico/yubikey-manager/archive/refs/tags/{{ version }}.tar.gz
+  ref: v{{version}}}
+
+display-name: yubikey-manager
+
+versions:
+  github: Yubico/yubikey-manager
+
+dependencies:
+  python.org: '^3'
+  linux:
+    pcsclite.apdu.fr: '^2'
+
+build:
+  dependencies:
+    pip.pypa.io: '*'
+    swig.org: '*'
+  script: |
+    pip install install --prefix={{prefix}} yubikey-manager
```

Comment:
this looks odd (`install install`?). if `yubikey-manager` uses a venv, consider the python-venv.sh helper. my guess, from the errors, is that it's not installing everything needed to the prefix.

Reply:
> this looks odd (`install install`?)

good catch